utils/linux/proc_events.py

```diff
@@ -206,9 +210,11 @@ def register_exit_callback(callback: Callable):
 
     The callback should receive three arguments: tid, pid and exit_code.
     """
+    _proc_events_listener: _ProcEventsListener
```

Comment:
Instead of re-defining its type, the better thing to do here IMO is to `assert _proc_events_listener is not None`. This also teaches mypy

Reply:
Cool, didn't know that